Makefile

```diff
@@ -12,11 +12,8 @@ client="mock"
 chosen_tests=""
 responses="pass"
 
-check-dangerous-indexing:
-	bash tests/check-dangerous-index.sh
 
-
```

Comment:
i removed this (even though it's useful) because it doesn't work with the ABFE class